src/graphql/account.rs

```diff
@@ -554,14 +555,14 @@ struct AccountTotalCount;
 
 #[Object]
 impl AccountTotalCount {
-    /// The total number of edges.
-    async fn total_count(&self, ctx: &Context<'_>) -> Result<usize> {
+    /// The total number of edges in string within the representable range of `usize`
+    async fn total_count(&self, ctx: &Context<'_>) -> Result<StringNumber<usize>> {
```

Comment:
There is no need to change the part used for paging.

Reply:
Done, removed `StringNumber` from paging for `clusterTotalCount` and `accountTotalCount`.